rkt/api_service.go

```diff
@@ -51,7 +51,7 @@ var (
 
 func init() {
 	cmdRkt.AddCommand(cmdAPIService)
-	cmdAPIService.Flags().StringVar(&flagAPIServiceListenClientURL, "--listen-client-url", common.APIServiceListenClientURL, "address to listen on client API requests")
```

Comment:
@joshix I tend to change the flag to take just an integer for the PORT, like what we did in the metadata service. How does that sound? (Sure it will break backward compatibility, but we are able to do so as the api-service is still in experimental stage)